app/views/groups/members/index.html.erb

```diff
@@ -14,18 +14,91 @@
   <% end %>
 <% end %>
 
-<%= turbo_frame_tag "new_member_modal" %>
 
 <div class="bg-white dark:bg-slate-800">
+  <%= turbo_frame_tag "new_member_modal" %>
+  <%= turbo_frame_tag("member-update-alert") %>
+  <ul class="flex flex-wrap -mb-px">
+    <li class="mr-2">
+      <%= link_to t(:"groups.members.index.tabs.members"),
+      group_members_path(),
+      data: {
+        turbo_frame: "_top"
+      },
+      class:
+        (
+          if params[:tab] == "invited_groups"
+            "inline-block p-4 border-b-2 border-transparent rounded-t-lg hover:text-gray-600 hover:border-gray-300 dark:hover:text-gray-300"
+          else
+            "inline-block p-4 text-blue-600 border-b-2 border-blue-600 rounded-t-lg active dark:text-blue-500 dark:border-blue-500"
+          end
+        ),
+      aria: {
+        current: params[:tab] == "invited_groups" ? "false" : "page"
+      } %>
+    </li>
+    <li class="mr-2">
+      <%= link_to t(:"groups.members.index.tabs.groups"),
+      group_members_path(tab: "invited_groups"),
+      data: {
+        turbo_frame: "_top"
+      },
+      class:
+        (
+          if params[:tab] == "invited_groups"
+            "inline-block p-4 text-blue-600 border-b-2 border-blue-600 rounded-t-lg active dark:text-blue-500 dark:border-blue-500"
+          else
+            "inline-block p-4 border-b-2 border-transparent rounded-t-lg hover:text-gray-600 hover:border-gray-300 dark:hover:text-gray-300"
+          end
+        ),
+      aria: {
+        current: params[:tab] == "invited_groups" ? "page" : "false"
+      } %>
+    </li>
+  </ul>
   <div class="flex flex-col">
-    <div class="overflow-x-auto">
-      <div class="shadow">
-        <%= turbo_frame_tag("member-update-alert") %>
-          <%= turbo_frame_tag "group_members_list",
-          src: group_members_path(@group, format: :turbo_stream, page: params[:page]),
-          loading: :lazy %>
+    <%= turbo_frame_tag("invited-group-alert") %>
+    <%= turbo_frame_tag "members",
+    src:
+      (
+        if params[:tab] == "invited_groups"
+          group_group_links_path(
+            format: :turbo_stream
+          )
+        else
+          group_members_path(
+            format: :turbo_stream
+          )
+        end
+      ),
+    loading: :lazy do %>
+      <table class="min-w-full table-fixed dark:divide-gray-600">
+        <tbody
+          class="
+            bg-white
+            divide-y
+            divide-gray-200
+            dark:bg-gray-800
+            dark:divide-gray-700
+          "
+        >
+          <% 15.times do %>
+            <tr>
+
+              <td class="p-4 animate-pulse">
```

Comment:
**thought (non-blocking):** This is definitely not for this PR to consider and perhaps @ericenns wants to weigh in on this.  I find these skeletons really annoying.  They cause a bunch of page flicker that does not really seem necessary with the speed that we get from this (although that may be different in production).  Maybe we should just have a loading message with a spinner if it takes longer or something that does not create so much flicker.